### DISCUSSION TITLE
GUI Parallel plotting is twice as fast as the Queue Trick

### DISCUSSION THREAD

**Discussion by Teledyol** [Chia-Network/chia-blockchain#3501] [General]
Before I finally got a new set of RAM (4x32) in my Mac Pro Trashcan, I was getting fairly good results using the Queue Trick of setting up several Queue plots with different names.

Once I got the new RAM, it was about the same time as we started using 1.0.4. (the switch was a bit wacky and stepped on one sequence)

I went through a couple failed attempts at setting up four queues at 6750 RAM. So, I decided to just set up the old GUI "plot in parallel" with the four. It ran great, with fewer errors on the log. I got for at an average of 5.4 Hours each

So I'm trying to kick back into the Queue Trick again and currently running much slower with the average at 7.5 hours each with the GUI showing 65% completion (prolly inaccurate). This is comparable to the first two failed attempts. 

I had a lot of errors up to midnight PST last night and it's now almost 8:30, without errors

Is anyone else showing slower plotting using the Queue trick?

**Comment by ful188** [comment]
me to

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
